### DISCUSSION TITLE
[12.10.2024] Blog 1 - Projekt & Team Vorstellung

### DISCUSSION THREAD

**Discussion by Kartoffeleintopf** [bunki1224/Softwareengineering-Projekt#5] [Show and tell]
## 1 - Unser Team
Unser Team besteht aus drei Entwicklern, die sich der Entwicklung der Travel Planner App verschrieben haben. bunki1224, übernimmt neben der Rolle des Entwicklers die des Projektmanagers. TimHitschfeld, der ebenfalls als Entwickler tätig ist. Und dann noch Kartoffeleintopf welcher das Entwicklerteam komplettiert.
## 2 - Die Vision
Die Vision hinter der Travel Planner App ist es, Nutzern eine effiziente und benutzerfreundliche Möglichkeit zu bieten, ihre Reisen zu planen und zu organisieren. Mit der App können Nutzer ihre Flüge, Hotels und Aktivitäten an einem einzigen Ort organisieren. Das Haupt Feature der App ist eine interaktive Timeline, die es den Nutzern ermöglicht, ihre Reise im Detail zu planen, indem sie ihre Reisedaten visuell anordnen und verfolgen. Flüge, Hotelbuchungen und Aktivitäten lassen sich chronologisch hinzufügen und bearbeiten, was die Reiseplanung erheblich vereinfacht.
## 3 - Der Techstack
Unser Techstack umfasst moderne Technologien, die eine robuste und skalierbare Anwendung ermöglichen. Für das Frontend verwenden wir React js, während das Backend mit Node js entwickelt wird. Docker sorgt für die Containerisierung, und Material UI dient als UI Framework. Für die Datenbank setzen wir auf MongoDB und Postgresql, während GitHub für die Versionskontrolle genutzt wird.
Zur Verwaltung des Projekts nutzen wir GitHub Projects, wo auch das Repository und der Blog zu finden sind. Die Plattform ermöglicht es uns, den Fortschritt des Projekts zu verfolgen und effizient zu verwalten.
## 4 - Scrum
Unser Scrum-Setup basiert ebenfalls auf GitHub Projects, wobei die Sprintlänge auf eine Woche festgelegt ist.
Der Screenshot unseres Kanban-Boards vom 12. Oktober 2024 zeigt den derzeitigen Stand des Projekts.

## 5 - Das Kanban Board
Screenshot Stand: 12.10.2024
![KanbanBoard](https://github.com/user-attachments/assets/c606806e-86a0-44eb-8ede-af2d87eb9bf6)

> Euer Trip Ahead Team
> CN

**Comment by Ninetilt** [comment]
Uns gefällt besonders die Idee der interaktiven Timeline zur Reiseplanung. Diese Funktion, mit der Nutzer Flüge, Hotels und Aktivitäten chronologisch anordnen und bearbeiten können, hört sich unglaublich hilfreich für Reisegruppen an. Eine visuelle Darstellung der Reisedaten bietet einen echten Mehrwert und vereinfacht die Planung erheblich. 

Viele Grüße
Team TDC (JS)

**Comment by carinabsn** [comment]
Wir vom GymTracker X-Team sind begeistert von eurer Projekt-Vision, besonders eure innovativen Ideen wie z.B. die interaktive Timeline, machen euren Projekt zu einem der Top-Gesprächsthemen in unseren kleinen Pausen.
Mit Spannung und Vorfreude werden wir euren weiteren Entwicklungsverlauf verfolgen und hoffen wir können noch viel von eurer Kreativität lernen.

Beste Grüße
Euer GymTrackerX-Team

**Comment by schmid-di** [comment]
Euer Blog zeigt deutlich, dass ihr eine klare Vision für die Travel Planner App habt, was sehr motivierend wirkt. Besonders positiv finde ich eure Wahl des Techstacks, der modern und skalierbar ist, sowie euren Einsatz von Scrum, um den Projektfortschritt effizient zu steuern. Auch die Transparenz, mit der ihr über euren Prozess berichtet, schafft eine gute Grundlage für das Projekt.

Ein kleiner Verbesserungspunkt wäre, dass im Kanban-Board die zentralen Features der App, wie die interaktive Timeline, noch nicht sichtbar sind. Es könnte hilfreich sein, diese frühzeitig zu priorisieren, um sicherzustellen, dass sie in der Entwicklung nicht zu kurz kommen.

Euer UMOC Team
/JMS /PL
